web/src/stream_edit.js

```diff
@@ -596,7 +597,13 @@ export function initialize() {
                 sub.stream_id,
             )}']`,
         );
-        stream_settings_components.sub_or_unsub(sub, $stream_row);
+
+        if ($target.hasClass("subscribe-button")) {
+            // Prevents displaying loading spinner in the stream row.
+            stream_settings_components.sub_or_unsub(sub);
+        } else {
+            stream_settings_components.sub_or_unsub(sub, $stream_row);
+        }
```

Comment:
Can you explain precisely what happens, ideally with a screenshot where you've added some artificial latency, in the left panel while waiting for the request to return? I'd like to make sure that left-panel state looks good with the spinner removed.

Reply:
If the `$stream_row` is passed in `sub_or_unsub`, it displays the spinner on that stream in left panel. Here, for the subscribe button, we are not passing it to not show spinner.

<details>
<summary>Screenshots (latency of 3 seconds)</summary>

With spinner (before):

![fix-jitter-loading](https://github.com/user-attachments/assets/960ee6d9-fd42-4f80-bd84-0fbed45caec0)

Spinner removed (after):

![fix-jitter-noloading](https://github.com/user-attachments/assets/f3d07bc4-7346-4385-aea4-36564e2602c3)

</details>